README.md

```diff
@@ -220,6 +221,20 @@ These licenses apply to the `HomeRacker - Core` system and customizable rackmoun
 
 HomeRacker is an unregistered trademark of Patrick Pötz (), first used publicly on 12.04.2025.
 
+# 🚀 Releases
+
+HomeRacker uses automated releases powered by Camunda's reusable workflows:
+
+- **Automated versioning**: Follows [Semantic Versioning](https://semver.org/) based on [Conventional Commits](https://www.conventionalcommits.org/)
+- **Changelog generation**: Automatically generated from commit messages
+- **Release schedule**: Weekly auto-merge of release PRs every Monday at 9:00 AM UTC (or manual trigger)
```

Comment:
also leave out the exact time. rather go with weekly. people can look it up in the workflow itself.